web/src/engine/websites/Toonkor.ts

```diff
@@ -0,0 +1,43 @@
+﻿import { Tags } from '../Tags';
+import icon from './Toonkor.webp';
+import { DecoratableMangaScraper } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import * as CoreView from './decorators/CoreView';
+
+function ChapterExtractor(element: HTMLElement) {
+    return {
+        id: element.dataset.role,
+        title: element.textContent.trim()
+    };
+}
+
+const pageScript = `
+    new Promise (resolve => {
+        const dom =  new DOMParser().parseFromString(toon_img, 'text/html');
+        resolve([...dom.querySelectorAll('img')].map(img => new URL(img.src, window.location.origin).href));
+    })
+`;
+
+@Common.MangaCSS(/https:\/\/toonkor\d+\.com\/[^/]+$/, 'table.bt_view1 td.bt_title')
+@CoreView.MangasSinglePageCSS(['/웹툰/연재?fil=제목', '/웹툰/완결?fil=제목'], 'div.section-item-title a#title', Common.AnchorInfoExtractor())
```

Comment:
This is misleading, Toonkor is not related to CoreView.
If this is going to be used as a more generic decorator, it should be moved out of CoreView.

Reply:
Exactly what i thought. I'll do that soon